src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -141,6 +146,7 @@ export const getContainerNameFromUrl = (containerUrl: string) => {
 };
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186236868-43ec87e2-89e7-4696-bb14-7130343f0a37.png)

Reply:
had commented and marked as "resolved" this on the original PR